common/src/main/java/com/wfsample/common/BeachshirtsUtils.java

```diff
@@ -0,0 +1,66 @@
+package com.wfsample.common;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLParser;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+
+import javax.annotation.Nullable;
+
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Utilities for use by the various beachshirts application related services.
+ *
+ * @author Srujan Narkedamalli (snarkedamall@wavefront.com).
+ */
+public final class BeachshirtsUtils {
```

Comment:
Nit - You have a mix of BeachShirtsWebResource and Beachshirt

I suggest sticking to BeachShirt everywhere in the code for capital `B`. 

For lowercase, let's stick to beachshirt instead of beachShirt.